src/main/java/racingcar/domain/RaceRecords.java

```diff
@@ -0,0 +1,53 @@
+package racingcar.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class RaceRecords {
+
+    private final List<RaceRecord> raceRecords;
+
+    public RaceRecords() {
+        this.raceRecords = new ArrayList<>();
+    }
+
+    public List<RaceRecord> getRaceRecords() {
+        return raceRecords;
+    }
+
+    public RaceRecord saveRecords(Cars cars, int round) {
+        Map<String, Integer> map = new HashMap<>();
+        for (Car car : cars.getCars()) {
+            map.put(car.getName(), car.getLocation());
+        }
+        return new RaceRecord(round, map);
+    }
+
+    public List<String> getWinner() {
+        RaceRecord lastRaceRecord = getLastRaceRecord();
+        Map<String, Integer> recordsWithCarName = lastRaceRecord.getRecordsWithCarName();
+        Integer max = Collections.max(recordsWithCarName.values());
+        return getWinnerNames(recordsWithCarName, max);
+    }
+
+    private RaceRecord getLastRaceRecord() {
+        return getRaceRecords().get(getRaceRecords().size() - 1);
+    }
+
+    private List<String> getWinnerNames(Map<String, Integer> map, Integer max) {
+        List<String> winners = new ArrayList<>();
+        for (String name : map.keySet()) {
+            if (map.get(name).equals(max)) {
+                winners.add(name);
+            }
+        }
+        return winners;
+    }
+
+    public void saveSingleRoundRecord(int round, Cars cars) {
+        getRaceRecords().add(saveRecords(cars, round));
```

Comment:
```suggestion
        raceRecords.add(saveRecords(cars, round));
```
getRaceRecords()를 사용할 필요가 있을까요?
오히려 메서드를 확인해야 해서 가독성이 떨어지는거 같은데 재희님의 생각은 어떠신가요? 😄